Update Torch-DirectML samples and docs for Torch-DirectML 2.3.0 (#610)
microsoft/DirectML: PyTorch/diffusion/sd/README.md

# Stable Diffusion Turbo & XL Turbo
This sample provides a simple way to load and run Stability AI's text-to-image generation models, Stable Diffusion Turbo & XL Turbo, with our DirectML-backend.

- [About the Models](#about-the-models)
- [Setup](#setup)
- [Run the App](#run-the-app)
- [External Links](#external-links)
- [Model License](#model-license)

## About the Models

> Stable Diffusion Turbo & XL Turbo are distilled versions SD 2.1 and SDXL 1.0 respectively. Both models are fast generative text-to-image model that can synthesize photorealistic images from a text prompt in a single network evaluation.

Reviewer: ```suggestion
Stable Diffusion Turbo & XL Turbo are distilled versions of SD 2.1 and SDXL 1.0 respectively. Both models are fast generative text-to-image model that can synthesize photorealistic images from a text prompt in a single network evaluation.
```
missing word

Refer to the HuggingFace repositories for [SDXL Turbo](https://huggingface.co/stabilityai/sdxl-turbo) and [SD Turbo](https://huggingface.co/stabilityai/sd-turbo) for more information.

## Setup

> Once you've setup `torch-directml` following our [Windows](https://learn.microsoft.com/en-us/windows/ai/directml/pytorch-windows) and [WSL](https://learn.microsoft.com/en-us/windows/ai/directml/pytorch-wsl) guidance, install the requirements by running:

Reviewer: ```suggestion
Once you've set up `torch-directml` following our [Windows](https://learn.microsoft.com/en-us/windows/ai/directml/pytorch-windows) and [WSL](https://learn.microsoft.com/en-us/windows/ai/directml/pytorch-wsl) guidance, install the requirements by running:
```
Setup is a noun. Set up is the verb.

```
pip install -r requirements.txt
```

## Run the App
To use Stable Diffusion with the text-to-image interface, run:
```bash
> python app.py
```

When you run this code, a local URL will be displayed on the console. Open http://localhost:7860 (or the local URL you see) in a browser to interact with the text-to-image interface.

> Within the interface, use the dropdown to switch between SD Turbo and SDXL Turbo. You can also use the slider to set the number of iteration steps, 1 to 4, for image generation.

Reviewer: ```suggestion
Within the interface, use the dropdown to switch between SD Turbo and SDXL Turbo. You can also use the slider to set the number of iteration steps (1 to 4) for image generation.
```
Ignorable, but slightly smoother read.
